mixer/pkg/lang/externs_test.go

```diff
@@ -431,12 +431,9 @@ func TestExternMatches(t *testing.T) {
 	}
 
 	for _, c := range cases {
-		m, err := externMatches(c.p, c.s)
+		_, err := externMatches(c.p, c.s)
 		if err != nil {
 			t.Fatalf("Unexpected error: %+v, %v", c, err)
-			if m != c.e {
```

Comment:
This doesn't seem right?

Reply:
Rejiggered. The original use of t.Fatalf was incorrect, leading to my original refactor. Replacing Fatalf with Errorf makes things make sense now.